tools/lint.js

```diff
@@ -24,7 +24,8 @@ if (Deno.args.includes("--rs")) {
 
 if (!didLint) {
   await dlint();
-  await dlintPreferPrimordials();
+  // todo(dsherret): re-enable
+  // await dlintPreferPrimordials();
```

Comment:
There were over 200 errors, which would be too much to update in this PR. I'll open an issue after.

Reply:
Sounds good, but we should make it pass linter ASAP. @petamoriken could you advise here?

Reply:
I opened https://github.com/denoland/deno/issues/17709